src/pages/Agency/Agency.js

```diff
@@ -1,10 +1,12 @@
 import React, { Component } from 'react'
+import CreateAgencyBtn from "../../components/CreateAgencyBtn/CreateAgencyBtn";
 
 export default class Agency extends Component {
```

Comment:
I think Nick suggested keeping `export` statements at the end of files, so we should stick to that